### PR TITLE
Remove broken links to Portage Network pages #339  #trivial 

### PR DESCRIPTION
**Pull request by 200455939-yashu**
Fixes # .

Issues with the Links resolved

Changes proposed in this PR: [339](https://github.com/portagenetwork/roadmap/issues/339)
-

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -86,10 +86,12 @@ def how_to_manage_your_data_path
...
   def contacts_at_your_instutution_path
     if I18n.locale == 'fr_CA'
-      'https://portagenetwork.ca/fr/outils-et-ressources/personnes-ressources-pour-la-gdr-dans-les-etablissements/'
+      # 'https://portagenetwork.ca/fr/outils-et-ressources/personnes-ressources-pour-la-gdr-dans-les-etablissements/'
+      'https://alliancecan.ca/fr/services/gestion-des-donnees-de-recherche/apprentissage-et-ressources/personnes-ressources-dans-les-etablissements'
     else
       # Handling "en_CA" locale
-      'https://portagenetwork.ca/tools-and-resources/institutional-rdm-contacts/'
+      # 'https://portagenetwork.ca/tools-and-resources/institutional-rdm-contacts/' #OLD URK
+      'https://alliancecan.ca/en/services/research-data-management/learning-and-training/institutional-contacts'
```
Comment (lagoan) [inline app/helpers/application_helper.rb]: We don't need to keep the old URLs now so we can remove these commented out code.

**Review by lagoan — CHANGES_REQUESTED** [review]
This is good work. We just need to remove those old links since we don't want old code to stay for long. If we are not using it we should remove it.

With any changes to text, we need to run the translation worflow. We will talk about that in a bit.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Unreleased
```
Comment (lagoan) [inline CHANGELOG.md]: All looks good! Just make sure to add an entry to CHANGELOG. This is a minor change and we need to keep a track of our changes.

**Review by lagoan — CHANGES_REQUESTED** [review]
We just need an entry on the CHANGELOG file and we will be done with this PR! 😄

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (200455939-yashu) [reply app/helpers/application_helper.rb]: Updated this

**Review by lagoan — APPROVED** [review]
GW!

**Pull request merged**